src/components/ArraySchema.js

```diff
@@ -0,0 +1,92 @@
+import { Fragment } from 'react';
+import _ from 'lodash';
+import { Field } from 'react-final-form';
+import { FieldArray } from 'react-final-form-arrays';
+
+const styles = {
+  flexCenter: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  labelWidth: {
+    width: '200px',
+  },
+  centerItem: {
+    margin: '0 auto',
+  },
+};
+
+const ArraySchema = ({ inputList, path, title }) => {
+  const addFields = () => {
```

Comment:
When we do the refactor, I would like to put comments pretty much anywhere that is either extra confusing, or where we are doing something specific to `final-form` or another library we are using. Just to call out places where logic might be much different when built by someone else.